docs/getting-started/benchmark.md

```diff
@@ -0,0 +1,211 @@
+We used the same `c7gn.16xlarge` for running `memtier_benchmark` and we used the same commands
+to test writes, reads and pipelined reads:
+
+| Test          | Ops/sec   | Avg. Latency (us) | P99.9 Latency (us) |
+|---------------|-----------|-------------------|--------------------|
+| Write-Only    | 5.2M      | 250               | 631                |
+| Read-Only     | 6M        | 271               | 623                |
+| Pipelined Read| 8.9M      | 323               | 839                |
+
+
+### Writes
+`memtier_benchmark -s $SERVER_PRIVATE_IP --distinct-client-seed --hide-histogram --ratio 1:0 -t 60 -c 20 -n 200000`
+
+```
+============================================================================================================================
+Type         Ops/sec     Hits/sec   Misses/sec    Avg. Latency     p50 Latency     p99 Latency   p99.9 Latency       KB/sec
+----------------------------------------------------------------------------------------------------------------------------
+Sets      5195097.56          ---          ---         0.26012         0.24700         0.49500         0.63100    400230.15
+```
+
+### Reads
+`memtier_benchmark -s $SERVER_PRIVATE_IP --distinct-client-seed --hide-histogram --ratio 0:1 -t 60 -c 20 -n 200000`
+```
+============================================================================================================================
+Type         Ops/sec     Hits/sec   Misses/sec    Avg. Latency     p50 Latency     p99 Latency   p99.9 Latency       KB/sec
+----------------------------------------------------------------------------------------------------------------------------
+Gets      6078632.89   6078632.89         0.00         0.27177         0.26300         0.49500         0.62300    438616.86
+```
+
+### Pipelined Reads
+`memtier_benchmark -s $SERVER_PRIVATE_IP --ratio 0:1 -t 60 -c 5  -n 200000  --distinct-client-seed --hide-histogram --pipeline=10`
+
+```
+============================================================================================================================
+Type         Ops/sec     Hits/sec   Misses/sec    Avg. Latency     p50 Latency     p99 Latency   p99.9 Latency       KB/sec
+----------------------------------------------------------------------------------------------------------------------------
+Gets      8975121.86   8975121.86         0.00         0.32325         0.31100         0.52700         0.83900    647619.14
+```
+
+
+## Comparison with Garnet
+Microsoft Research recently released Garnet ([https://github.com/microsoft/garnet]),
+a remote cache store. Due to interest within the Dragonfly community, we decided to compare
+Garnet's performance with Dragonfly's. This comparison focuses on performance results
+and does not delve into architectural differences or Redis compatibility implications.
+
+<i>Note: Unfortunately, Garnet does not have aarch64 build available,
```

Comment:
I think that the usage of `<i>` here messes up the formatting. Maybe use `_` instead, or only make the `Note:` part italic?